zulipterminal/ui_tools/views.py

```diff
@@ -1016,6 +1016,26 @@ def __init__(self, controller: Any, title: str, *, zt_version: str,
         super().__init__(controller, widgets, 'ABOUT', popup_width, title)
 
 
+class UserInfoView(PopUpView):
+    def __init__(self,
+                controller: Any,
+                display_data: dict,
```

Comment:
Can we improve this type?

Reply:
Definitely. I'll update `helper.py` as mentioned above :)